modules/ROOT/attachments/admin-api.yaml

```diff
@@ -1463,6 +1463,29 @@ paths:
         schema:
           type: integer
           format: int64
+  /debug/restart_service:
+    put:
+      tags: 
+        -  Debugging
+      summary: Restart service
+      description: Restart a Redpanda service.
+      operationId: restart_service
+      parameters:
+        - name: service
+          in: query
+          required: true
+          schema:
+            type: string
+          example: 'schema-registry'
```

Comment:
@BenPope Do we have a list of acceptable values for `service=`?

Reply:
Just these two: https://github.com/redpanda-data/redpanda/blob/v23.3.12/src/v/redpanda/admin/server.cc#L4318-L4321